pkg/http/client.go

```diff
@@ -207,6 +211,10 @@ type DialerSettings struct {
 	FallbackDelay time.Duration `cfg:"fallback_delay" default:"0s"`
 }
 
+type TracingSettings struct {
+	ForwardTraceId bool `cfg:"forward_trace_id" default:"false"`
```

Comment:
Do we wanna have `true` as default here?\
For me this looks more like a opt-out than a opt-in.